### ISSUE TITLE
Linked Parameters

### ISSUE THREAD

**Issue by baconpaul** [surge-synthesizer/surge#1410]
In #1398 @itsmedavep suggested a way to link parameters so dragging one would drag the other. Useful for things like a paired modulator or a mixer section.

This is a rather difficult thing to do, but a good idea, so rather than lose it with that issue (where I did the main thing the issue was about) I split that request over here with a link back to his original commentary, and put this in the currently unscheduled milestone.

**Comment by VincyZed** [comment]
I agree, that would be great! This could also be a way to implement #1276 as well?

**Comment by baconpaul** [comment]
Oh yeah! That’s a slightly more complicated UI (since you have to pick the ‘other scene’ version) but it’s the exact same underlying mechanics. Definitely they are linked
